Gododir/gen.go

```diff
@@ -36,12 +37,13 @@ type mapping struct {
 }
 
 var (
+	start           time.Time
 	gopath          = os.Getenv("GOPATH")
 	sdkVersion      string
 	autorestDir     string
 	swaggersDir     string
 	testGen         bool
-	deps            do.S
+	deps            do.P
```

Comment:
This is the magic that makes it parallel?

Reply:
Yes